### ISSUE TITLE
classification, oriented bounding boxes, pose estimation, and instance segmentation请问预训练模型有了么，在哪里下载呢？

### ISSUE THREAD

**Issue by CachCheng** [ultralytics/ultralytics#19374]
### Search before asking

- [x] I have searched the Ultralytics [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar feature requests.

### Description

classification, oriented bounding boxes, pose estimation, and instance segmentation请问预训练模型有了么，在哪里下载呢？

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @CachCheng, thank you for your interest in Ultralytics 🚀! We recommend visiting the [Docs](https://docs.ultralytics.com/) for useful resources, where you’ll find details about model usage, pretrained weights, and downloads for tasks like classification, pose estimation, and instance segmentation. Check out the instructions for [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage as well.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a ❓ Question or feature request, kindly share additional context or elaborate on your needs so we can guide you effectively.

Join the Ultralytics community where it suits you best:
- For real-time discussions, visit [Discord](https://discord.com/invite/ultralytics) 🎧.
- Share or browse ideas on our [Discourse Forum](https://community.ultralytics.com/).
- Dive into threads and engage with other users on our [Subreddit](https://reddit.com/r/Ultralytics).

---

## Upgrade

Before proceeding, ensure that you are using the latest `ultralytics` version. Upgrade along with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) installed:

```bash
pip install -U ultralytics
```

---

## Environments

You can run YOLO in these verified environments (with CUDA, PyTorch, and dependencies preinstalled):

- **Notebooks**: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** VMs. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon AWS** AMIs. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Images**: See [Docker Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

---

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If the above badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. CI tests verify operations for YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu.

---

This is an automated response to assist you promptly, and an Ultralytics engineer will follow up with further support soon. 😊

**Comment by tsh96** [comment]
You can download pre-trained models from https://github.com/ultralytics/assets/releases/tag/v8.3.0
Pre-trained models for classification, oriented bounding box, pose estimation, and instance segmentation in yolo12 are currently not available.

**Comment by glenn-jocher** [comment]
Ultralytics YOLO12 models for classification, oriented bounding boxes (OBB), pose estimation, and instance segmentation are currently in development and not yet released as pretrained models. For these tasks, we recommend using our latest available YOLO11 models which offer pretrained weights for [instance segmentation](https://docs.ultralytics.com/models/yolo11/#segmentation), [classification](https://docs.ultralytics.com/models/yolo11/#classification), and [pose estimation](https://docs.ultralytics.com/models/yolo11/#pose-estimation). 

You can download YOLO11 pretrained models from our official repository at [Ultralytics YOLO11 Models](https://github.com/ultralytics/ultralytics#models). For commercial use, please ensure compliance with our licensing requirements outlined at [Ultralytics Licensing](https://ultralytics.com/license).
